src/routes/__error.svelte

```diff
@@ -8,34 +8,52 @@
 </svelte:head>
 
 <section>
-  <img src="/assets/png/404-h.png" class="wide" alt="404 - Page Not Found" />
-  <img src="/assets/png/404-v.png" class="mobile" alt="404 - Page Not Found" />
+  <em>404</em>
+  <h1>Frank can't find where you're going!</h1>
+  <img src="/assets/png/lost-frank.png" class="wide" alt="404 - Page Not Found" />
+  <img src="/assets/png/lost-frank.png" class="mobile" alt="404 - Page Not Found" />
 </section>
 
 <style>
   section {
     min-height: 100vh;
     display: flex;
+    flex-direction: column;
     justify-content: center;
     align-items: center;
     margin: 0 24px;
+    color: var(--acm-light);
   }
-
-  section .mobile {
-    display: none;
+  section h1 {
+    margin-bottom: 2em;
+    align-items: center;
+    text-align: center;
+  }
+  section em {
+    align-items: center;
+    text-align: center;
+    font-size: 50px;
+    margin-bottom: 0.5em;
+    margin-top: 2em;
   }
 
   section img {
     width: 100%;
     max-width: 1000px;
     height: auto;
+    margin-bottom: 3em;
+  }
+
+  @media (min-width: 900px) {
+    section .mobile {
+      display: none;
+    }
   }
```

Comment:
Unless I misunderstand, the `wide` and `mobile` class names seem to not affect anything. If so, you can go ahead and remove these lines of CSS.
```suggestion
```